cmake/dependencies.cmake

```diff
@@ -67,22 +70,17 @@ find_package(tbb)
 ##########################
 #         boost          #
 ##########################
-find_package(Boost 1.65.0 REQUIRED
-    COMPONENTS
+set(BOOST_COMPONENTS
     filesystem
     system
     )
-add_library(boost INTERFACE IMPORTED)
-set_target_properties(boost PROPERTIES
-    INTERFACE_INCLUDE_DIRECTORIES ${Boost_INCLUDE_DIRS}
-    INTERFACE_LINK_LIBRARIES "${Boost_LIBRARIES}"
+hunter_add_package(Boost COMPONENTS
+    ${BOOST_COMPONENTS}
+    )
+find_package(Boost 1.65.1 CONFIG REQUIRED
```

Comment:
Also, boost and hunter is installing into home directory: `$HOME/.hunter/...`. I think need change it to `external` folder.

Reply:
Also, what do u think about downloading only a part of the boost? Now for each clean build, there is need to download 80+ MB of boost. Maybe we may reduce it?

Reply:
> Also, what do u think about downloading only a part of the boost? Now for each clean build, there is need to download 80+ MB of boost. Maybe we may reduce it?

IDK, because I use a standard components-choosing mechanism of Hunter: required components explicitly marked in __hunter-config.cmake__ and __dependencies.cmake__

Reply:
>Need to add functionality to set a path for the boost directly if it installed in the system.


it is already there, just pass BOOST_ROOT to cmake:
`cmake .. -DBOOST_ROOT=$path`